db/db_properties_test.cc

```diff
@@ -615,8 +615,9 @@ TEST_F(DBPropertiesTest, NumImmutableMemTable) {
     writeOpt.disableWAL = true;
     options.max_write_buffer_number = 4;
     options.min_write_buffer_number_to_merge = 3;
-    options.max_write_buffer_number_to_maintain = 4;
-    options.write_buffer_size = 1000000;
+    options.write_buffer_size = 2000000;
+    options.max_write_buffer_size_to_maintain =
+        4 * static_cast<int64_t>(options.write_buffer_size);
```

Comment:
reset write_buffer_size to 1MB in latest commit. Also the logic is a bit different because previously we count the total number of immutable memtables (both flushed and unflushed) while now it's size of mutable and immutable memtables